message-me/app/assets/stylesheets/custom.css.scss

```diff
@@ -13,3 +13,9 @@
   height: 15em;
   overflow: auto;
 }
+
+.input {
+  button {
+    display: none;
+  }
```

Comment:
Whats this?

Reply:
To make the button invisible. But still, need the button in order to call the `.click()` method on it when submitting then clear the input

Reply:
Why do you need a button?

Reply:
eh. hello. you remove the button. then why still have this?
```
.input
```
is so misleading. its like its the input but its actually the input container